documentation/docs/06-form-actions.md

```diff
@@ -255,6 +255,8 @@ Without an argument, `use:enhance` will emulate the browser-native behaviour, ju
 - call `goto` on a redirect response
 - render the nearest `+error` boundary if an error occurs
 
+> By default the `form` property is only updated for actions that are in `+page.server.js` alongside `+page.svelte` because in the native form submission case you would be redirected to the page the action is on
```

Comment:
missing some articles
```suggestion
> By default the `form` property is only updated for actions that are in a `+page.server.js` alongside the `+page.svelte` because in the native form submission case you would be redirected to the page the action is on
```